.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,38 @@
+name: ⚒️ WiX Improvement Proposal
```

Comment:
WIP or feature request?

Reply:
I merged them. I know it creates more work when requesting features, but several times now, I've wished that a feature request was formatted as a WIP. Making someone walk through the whole process when asking for a feature request is a good thing.

I recognize that this creates more work for us (maintainers on the project) when we're willing to bypass a lot of the extra detail because "we just know how to implement it" (same for bugs), but I think the tradeoff is still worth it (at least, at this moment).